jumpcloud-ADMU/Powershell/Public/Start-Migration.ps1

```diff
@@ -145,15 +145,15 @@ Function Start-Migration {
         $AGENT_INSTALLER_URL = "https://cdn02.jumpcloud.com/production/jcagent-msi-signed.msi"
         $AGENT_INSTALLER_PATH = "$windowsDrive\windows\Temp\JCADMU\jcagent-msi-signed.msi"
         $AGENT_CONF_PATH = "$($AGENT_PATH)\Plugins\Contrib\jcagent.conf"
-        $admuVersion = '2.8.5'
+        $admuVersion = '2.8.6'
         # Log Windows System Version Information
         Write-ToLog -Message:("OSName: $($systemVersion.OSName), OSVersion: $($systemVersion.OSVersion), OSBuildNumber: $($systemVersion.OsBuildNumber), OSEdition: $($systemVersion.WindowsEditionId)")
-
+        $script:JumpCloudUserID = $JumpCloudUserID
```

Comment:
scoping the script variable to prevent $null values from throwing: `The variable cannot be validated because the value  is not a valid value for the JumpCloudUserId variable.`